protocols/bftcosi/packets.go

```diff
@@ -4,10 +4,24 @@ import (
 	"crypto/sha512"
 	"errors"
 
+	"github.com/dedis/cothority/network"
 	"github.com/dedis/cothority/sda"
 	"github.com/dedis/crypto/abstract"
 )
 
+func init() {
+	for _, i := range []interface{}{
+		BFTSignature{},
+		Announce{},
+		Commitment{},
+		ChallengePrepare{},
+		ChallengeCommit{},
+		Response{},
+	} {
+		network.RegisterPacketType(i)
```

Comment:
Should we add a `network.RegisterPacketTypes`? Like there are other `Register*s`-functions?